### PR TITLE
Add a variety of new instants, sorceries and lands

### PR DESCRIPTION
**Pull request by brooks42**
Tried to add some more generic spells with mild pokemon themes

The Mysterious Feather is supposed to be the Ho-oh sighting at the end of the first episode 

The idea of the PC lands is to be storage lands themed around the pokemon storage system, like they're the lands pokemon go when you put them in storage. Was thinking of theming them a bit more, like have them require you to reveal an on-color pokemon or they're tapped, or put storage counters on them equal to the number of that color pokemon you control. Definitely open to thoughts about it

### CODE REVIEW

File: ptcg_cockatrice.xml [hunk 1]
```diff
@@ -1440,6 +1497,117 @@
...
+			<text>Tap all Pokémon target player controls. Those Pokémon don't untap during that player's next untap step.</text>
+		</card>
+		<card>
+			<name>Smog</name>
+			<set picURL="https://images.pokemontcg.io/base1/72_hires.png" rarity="common">PTCG</set>
+			<color>B</color>
+			<manacost>1BBB</manacost>
+			<type>Instant</type>
+			<tablerow>3</tablerow>
+			<text>Put a -1/-1 counter on all attacking Pokémon. Each player gets a poison counter.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1534]:
```suggestion
			<text>Put a -1/-1 counter on each attacking Pokémon. Each player gets a poison counter.</text>
```

File: ptcg_cockatrice.xml [hunk 2]
```diff
@@ -1440,6 +1497,117 @@
...
+			<text>Fire Spin deals 2 damage to target Pokémon. If that Pokémon would die this turn, exile it instead.</text>
+		</card>
+		<card>
+			<name>Gust</name>
+			<set picURL="https://images.pokemontcg.io/base1/72_hires.png" rarity="common">PTCG</set>
+			<color>WG</color>
+			<manacost>{W/G}</manacost>
+			<type>Instant</type>
+			<tablerow>3</tablerow>
+			<text>Gust deals 2 damage to target Pokémon with flying. If that Pokémon is attacking, Gust deals 5 damage, instead.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1580]:
```suggestion
			<text>Gust deals 2 damage to target Pokémon with flying. If that Pokémon is attacking, Gust deals 5 damage instead.</text>
```

File: ptcg_cockatrice.xml [hunk 3]
```diff
@@ -1541,6 +1709,43 @@
...
 At the beginning of your upkeep, return all cards you own exiled with Pokémon Storage System to your hand, then draw a card.</text>
 		</card>
+		<card>
+			<name>Town Map</name>
+			<set picURL="https://images.pokemontcg.io/sm12/197_hires.png" rarity="common">PTCG</set>
+			<color></color>
+			<manacost>1</manacost>
+			<type>Artifact</type>
+			<tablerow>1</tablerow>
+			<text>{2}, {T}, Sacrifice Town Map: Search your library for a land card, reveal it, and put it into your hand. Then shuffle your library.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1719]:
```suggestion
			<text>{2}, {T}, Sacrifice Town Map: Search your library for a land card, reveal it, and put it into your hand. Then shuffle.</text>
```

File: ptcg_cockatrice.xml [hunk 4]
```diff
@@ -1715,13 +1920,53 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+		</card>
+		<card>
+			<name>Teleport</name>
+			<set picURL="https://images.pokemontcg.io/ex3/90_hires.png" rarity="uncommon">PTCG</set>
+			<color>UB</color>
+			<manacost>1{U/B}</manacost>
+			<type>Enchantment - TM</type>
+			<tablerow>1</tablerow>
+			<text>Enchant Pokémon
+				Enchanted Pokémon has "{T}: Return this Pokémon and all TMs attached to it to their owners' hand."</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1958]:
```suggestion
				Enchanted Pokémon has "{T}: Return this Pokémon and all TMs attached to it to its owner's hand."</text>
```

File: ptcg_cockatrice.xml [hunk 5]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
 			<tablerow>0</tablerow>
 			<text>{T}, Sacrifice Safari Zone: Search your library for a basic land card, put it onto the battlefield tapped, then shuffle.</text>
 		</card>
+		<card>
+			<name>PC Grassland</name>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2095]:
```suggestion
			<name>Bill's Grassland</name>
```

File: ptcg_cockatrice.xml [hunk 6]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>PC Grassland enters the battlefield tapped.
+				{T}: Put a storage counter on PC Grassland.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2102]:
```suggestion
				{T}: Put a storage counter on Bill's Grassland.
```

File: ptcg_cockatrice.xml [hunk 7]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}, Remove any number of storage counters from PC Grassland: Add {W} for each storage counter removed this way.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2103]:
```suggestion
				{T}, Remove any number of storage counters from Bill's Grassland: Add {W} for each storage counter removed this way.</text>
```

File: ptcg_cockatrice.xml [hunk 8]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+		</card>
+		<card>
+			<name>PC Volcano</name>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2106]:
```suggestion
			<name>Bill's Volcano</name>
```

File: ptcg_cockatrice.xml [hunk 9]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>PC Volcano enters the battlefield tapped.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2112]:
```suggestion
			<text>Bill's Volcano enters the battlefield tapped.
```

File: ptcg_cockatrice.xml [hunk 10]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
 			<text>{T}, Sacrifice Safari Zone: Search your library for a basic land card, put it onto the battlefield tapped, then shuffle.</text>
 		</card>
+		<card>
+			<name>PC Grassland</name>
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>PC Grassland enters the battlefield tapped.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2101]:
```suggestion
			<text>Bill's Grassland enters the battlefield tapped.
```

File: ptcg_cockatrice.xml [hunk 11]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+		</card>
+		<card>
+			<name>PC Volcano</name>
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>PC Volcano enters the battlefield tapped.
+				{T}: Put a storage counter on PC Volcano.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2113]:
```suggestion
				{T}: Put a storage counter on Bill's Volcano.
```

File: ptcg_cockatrice.xml [hunk 12]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}, Remove any number of storage counters from PC Volcano: Add {R} for each storage counter removed this way.</text>
+		</card>
+		<card>
+			<name>PC Rainforest</name>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2117]:
```suggestion
			<name>Bill's Rainforest</name>
```

File: ptcg_cockatrice.xml [hunk 13]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>PC Rainforest enters the battlefield tapped.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2123]:
```suggestion
			<text>Bill's Rainforest enters the battlefield tapped.
```

File: ptcg_cockatrice.xml [hunk 14]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}: Put a storage counter on PC Rainforest.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2124]:
```suggestion
				{T}: Put a storage counter on Bill's Rainforest.
```

File: ptcg_cockatrice.xml [hunk 15]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}, Remove any number of storage counters from PC Rainforest: Add {G} for each storage counter removed this way.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2125]:
```suggestion
				{T}, Remove any number of storage counters from Bill's Rainforest: Add {G} for each storage counter removed this way.</text>
```

File: ptcg_cockatrice.xml [hunk 16]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+		</card>
+		<card>
+			<name>PC Bog</name>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2128]:
```suggestion
			<name>Bill's Bog</name>
```

File: ptcg_cockatrice.xml [hunk 17]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>PC Bog enters the battlefield tapped.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2134]:
```suggestion
			<text>Bill's Marsh enters the battlefield tapped.
```

File: ptcg_cockatrice.xml [hunk 18]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}: Put a storage counter on PC Bog.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2135]:
```suggestion
				{T}: Put a storage counter on Bill's Marsh.
```

File: ptcg_cockatrice.xml [hunk 19]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}, Remove any number of storage counters from PC Bog: Add {B} for each storage counter removed this way.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2136]:
```suggestion
				{T}, Remove any number of storage counters from Bill's Marsh: Add {B} for each storage counter removed this way.</text>
```

File: ptcg_cockatrice.xml [hunk 20]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+		</card>
+		<card>
+			<name>PC Beach</name>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2139]:
```suggestion
			<name>Bill's Coastline</name>
```

File: ptcg_cockatrice.xml [hunk 21]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>PC Beach enters the battlefield tapped.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2145]:
```suggestion
			<text>Bill's Coastline enters the battlefield tapped.
```

File: ptcg_cockatrice.xml [hunk 22]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}: Put a storage counter on PC Beach.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2146]:
```suggestion
				{T}: Put a storage counter on Bill's Coastline.
```

File: ptcg_cockatrice.xml [hunk 23]
```diff
@@ -1846,5 +2091,60 @@ At the beginning of your upkeep, return all cards you own exiled with Pokémon S
...
+				{T}, Remove any number of storage counters from PC Beach: Add {U} for each storage counter removed this way.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:2147]:
```suggestion
				{T}, Remove any number of storage counters from Bill's Coastline: Add {U} for each storage counter removed this way.</text>
```

**Comment by brooks42** [comment]
Merging without the Bill's comments, we'll just do those in XML

**Pull request merged**